### PR TITLE
Save directly after adding or editing a site

### LINKED ISSUE #15: New site not saved when quitting before logging out
Hello,
thank you very much for this great implementation of the mpw algorithm!

I noticed when using `qMasterPassword` on Linux with the Wayland compositor `sway`, that when quitting the application before pressing the "Logout" button, sites I added before get lost and are not visible anymore when I re-open qMasterPassword and re-login.

Can someone reproduce this? I'd be glad if I can be of help to debug this and thereby hopefully fix and improve the application.

### PR DESCRIPTION
**Pull request by teutat3s**
closes #15

### CODE REVIEW

File: src/main_window.cpp [hunk 1]
```diff
@@ -604,6 +604,7 @@ void MainWindow::addSite() {
 		edit_site.applyData();
 		m_current_user->getSites().append(site);
 		addSiteToUI(*site.get());
+                saveSettings();
```
Comment (bkueng) [inline src/main_window.cpp]: Can you use tabs instead of spaces for consistency?

**Review by bkueng — COMMENTED** [review]
Thanks!
Can you add it to `MainWindow::deleteSite` as well?

File: src/main_window.cpp (hunk 1 above)
Comment (teutat3s) [reply src/main_window.cpp]: Yeah sure, done.

**Comment by teutat3s** [comment]
> Thanks! Can you add it to `MainWindow::deleteSite` as well?

Done.

**Review by bkueng — APPROVED** [review]

**Pull request merged**